webapp/components/PostCard/index.vue

```diff
@@ -74,11 +82,16 @@ export default {
     HcRibbon,
     ContentMenu,
   },
+  mixins: [PostMutationHelpers],
   props: {
     post: {
       type: Object,
       required: true,
     },
+    width: {
+      type: Object,
+      default: () => {},
```

Comment:
Oh, ok, so `() => {}` creates an empty object. So, I guess the syntax for empty function is `() => () => {}` :thinking: